backend/src/modules/player-profiles/core/application/PlayerProfilesRepository.ts

```diff
@@ -0,0 +1,5 @@
+import {PlayerProfile} from "./command/PlayerProfile";
+
+export interface PlayerProfilesRepository {
+    findAll(): Promise<PlayerProfile[]>;
```

Comment:
Tutaj jest coś dziwnego, bo zrobiłeś to jako komendę. A nie zapisujemy każdej komendy tylko obiekt, który będzie rezultatem.